src/browser/components/pages/OverdueTaskDashboard/index.js

```diff
@@ -0,0 +1,18 @@
+import  React, { Component } from 'react'
+import OverdueTaskList from '../../molecules/OverdueTaskList/index'
+import { fetchURL, toStandardDate } from '../../../common/utils'
+import moment from 'moment'
+
+export default class OverdueTaskDashboard extends Component {
```

Comment:
I think we're overloading the use of the name `Dashboard`. Please change to `OverdueTasks`